### DISCUSSION TITLE
Multi-block select and Drag

### DISCUSSION THREAD

**Discussion by dfinster** [GitbookIO/community#242] [0. General bugs and questions] [answered]
When [the multi-block selection feature was announced](https://blog.gitbook.com/product-updates/new-in-gitbook-better-block-manipulation-and-private-content), it was explained that:
> When you have the blocks you need, you can drag them around your page

It's also demonstrated in the graphic on that page. 
![image](https://github.com/GitbookIO/community/assets/106555692/a2d48f97-299b-4a9c-81a3-d607e8de7b94)

This has been a huge productivity boost. However, recently this has stopped working. Now, when I drag multiple selected blocks with the mouse, the drag indicator says `Move block` instead of the expected `Move blocks` (plural), and only one of the selected blocks is moved. 

I see no mention of the drag function [in the documentation](https://docs.gitbook.com/content-creation/blocks#selecting-blocks-and-interacting-with-selected-blocks) for that feature. 

Was this feature removed, or is this a new bug?

**Comment by taszap** [comment, marked as answer]
Hey @dfinster 

Thanks for raising it! The feature was definitely not removed so it looks like a bug. I've raised this to our team.  
I will drop you an update here once we get to the bottom of this

**Reply by taszap** [reply]
Just a quick follow up this has been fixed and will be released this week, once again thank you for notifying us!

**Reply by dfinster** [reply]
Excellent work, thank you very much! I confirm it is working again for me when I used it today.
